src/schemes/solid/total_lagrangian_sph/system.jl

```diff
@@ -267,7 +275,7 @@ end
 
     @threaded system for particle in eachparticle(system)
         F_particle = deformation_gradient(system, particle)
-        pk1_particle = pk1_stress_tensor(F_particle, system)
+        pk1_particle = pk1_stress_tensor(F_particle, system, particle)
```

Comment:
Would it make sense to add `pk1_stress_tensor(system, particle)` to combine the two?
You may need to keep the original function as well for tests.